ledger/ledger-api-common/src/main/scala/com/digitalasset/ledger/api/validation/CommandsValidator.scala

```diff
@@ -75,6 +77,25 @@ final class CommandsValidator(ledgerId: LedgerId) {
         ),
       )
 
+  private def validateLedgerTime(
+      currentTime: Instant,
+      commands: ProtoCommands,
+  ): Either[StatusRuntimeException, Instant] = {
+    val minLedgerTimeAbs = commands.minLedgerTimeAbs
+    val minLedgerTimeRel = commands.minLedgerTimeRel
+
+    (minLedgerTimeAbs, minLedgerTimeRel) match {
+      case (None, None) => Right(currentTime)
+      case (Some(minAbs), None) =>
+        Right(currentTime.max(TimestampConversion.toInstant(minAbs)))
+      case (None, Some(minRel)) => Right(currentTime.plus(DurationConversion.fromProto(minRel)))
+      case (Some(_), Some(_)) =>
+        Left(
+          invalidArgument(
+            "min_ledger_time_abs can not be specified at the same time as min_ledger_time_rel"))
```

Comment:
`at the same time` may be confusing here. Maybe rewrite to `either min_ledger_time_abs or min_ledger_time_rel must be specified but not both`?